src/strategies/index.ts

```diff
@@ -735,6 +737,7 @@ const strategies = {
   'staked-uniswap-modifiable': stakedUniswapModifiable,
   'erc1155-balance-of-ids': erc1155BalanceOfIds,
   'erc1155-balance-of-ids-weighted': erc1155BalanceOfIdsWeighted,
+  'erc-1155-weighted-by-id': erc1155weighted,
```

Comment:
Thank you so much for explaining, can rename it to `erc1155-weighted-by-id` instead of `erc-1155-weighted-by-id` please :) that way it would be similar to other strategies names, everything else looks good 👍

Reply:
thanks @ChaituVR - updated